src/lib/widgets/episode/ui/episode-card.svelte

```diff
@@ -1,17 +1,28 @@
 <script lang="ts">
+  import type { Episode } from '$lib/shared/api';
   import { PlaybackButton } from '$lib/features/playback-controls';
   import { LikeButton } from '$lib/features/like-episode';
   import { DownloadLink } from '$lib/features/download-episode';
   import { EpisodeCardShell } from '$lib/entities/episode';
-  import type { Episode } from '$lib/shared/api';
+  import { paused } from '$lib/entities/audio';
+  import { addToListeningHistory } from '$lib/entities/audio/model/listening-history';
+  import { toggleGlobalPlayback } from '$lib/features/playback-controls';
```

Comment:
nitpick (non-blocking): this import could be merged with another import above from the same module